### ISSUE TITLE
Baresip container doesn't start

### ISSUE THREAD

**Issue by copiltembel** [baresip/docker#6]
Cool stuff, thank you for this. 

I have a problem though:

```
$ docker run ghcr.io/baresip/docker/baresip:v2.6.0
baresip v2.6.0 Copyright (C) 2010 - 2022 Alfred E. Heggestad et al.
conf: could not get config path: No such file or directory [2]
main: configure failed: No such file or directory [2]
ua: stop all (forced=1)
```

IIUC the config should be created automatically by the baresip exec when starting for the first time, but in this case is seems it cannot do that?

**Comment by sreimers** [comment]
Thanks for reporting, can you try the latest stable image. It should be fixed now by: 7ba564b61ae96c88001511a888855387868f1e9f

**Comment by copiltembel** [comment]
Works with stable version. Thank you!
